src/ODataHttpClient/ODataHttpClient.csproj

```diff
@@ -8,8 +8,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="[5.2.4,)" />
-    <PackageReference Include="Newtonsoft.Json" Version="[10.0.1,)" />
+    <PackageReference Include="Microsoft.AspNet.WebApi.Client" Version="[5.2.7,)" />
```

Comment:
Current:
Microsoft.AspNet.WebApi.Client >= 5.2.4

Your PR
Microsoft.AspNet.WebApi.Client >= 5.2.7

Would you tell me why you have raised the version range?

https://docs.microsoft.com/nuget/concepts/package-versioning#version-ranges